### PR TITLE
Adds orcs to the arcade

### PR DESCRIPTION
**Pull request by jcpirtle**

### CODE REVIEW

File: source/images/shapes/Used Images.txt
Comment (ehgoodenough) [inline source/images/shapes/Used Images.txt:2]: Smart :]

File: source/scripts/data/monsters.js
Comment (ehgoodenough) [inline source/scripts/data/monsters.js:21]: It's a stupid complaint, but could I get these in order from weakest to strongest?

Comment (ehgoodenough) [reply source/scripts/data/monsters.js:21]: It'll just make it easier to read from top to bottom.

**Comment by brandonhutchins** [comment]
Looks good to me.

**Comment by ehgoodenough** [comment]
@brandonhutchins: Could you add an "approval" to the github? You use the code review tools that Github just added.

**Comment by ehgoodenough** [comment]
An approval will be much more formalized than just a "LGTM" comment like we've been doing.

**Comment by jcpirtle** [comment]
Monster order has been updated

**Review by ehgoodenough — APPROVED** [review]
Approved!

**Comment by ehgoodenough** [comment]
I'm waiting on an approval from one of your peers to merge this into master.

**Comment by jcpirtle** [comment]
Brandon is here with me, how do you approve it. what is the tool name?

**Comment by ehgoodenough** [comment]
Go to the Files Changed tab to start a review. You can leave comments (like you should be doing during these reviews!!) or leave approvals.

**Review by brandonhutchins — APPROVED** [review]

**Pull request merged**
